.github/pull_request_template.md

```diff
@@ -1,66 +1,51 @@
-<!-- Start by describing the changes made in this Pull Request, and the reason for such changes. -->
+<!-- Please do not remove any information from this pull request. Instead, add N/A or leave blank if not applicable -->
 
-<!-- Reference any related issues or PRs here -->
+## What
 
 Fixes #
 
-<!-- Don't forget to update the title with something descriptive. -->
-<!-- If you can, add the appropriate labels -->
+## Why
 
-#### Accessibility
+<!-- Describe the reason for your changes. This will help the reviewer and future readers get additional context -->
 
-<!-- If you've changed or added any interactions, check off the appropriate items below. You can delete any that don't apply. Use this space to elaborate on anything if needed. -->
+## Screenshots or screencast
```

Comment:
I'm wondering if we should move this section below the "Testing Instructions" section. I've noticed that some PRs include screenshots beneath that section (for example: https://github.com/woocommerce/woocommerce-blocks/pull/10342), and it's really helpful when you're the Release Lead and can simply copy and paste the instructions along with the screenshots.

Reply:
Good suggestion @thealexandrelara ill make that change.